src/app/pages/send/send-crypto-asset-form/form/btc/btc-send-form.tsx

```diff
@@ -78,6 +79,11 @@ export function BtcSendForm() {
               <FormFooter balance={btcBalance.balance} />
               <HighFeeDrawer learnMoreUrl={HIGH_FEE_WARNING_LEARN_MORE_URL_BTC} />
               <Outlet />
+
+              {/* This is for testing purposes only, to make sure the form is ready to be submitted */}
```

Comment:
@kyranjamie @fbwoolf  not sure how to do it better. in tests we need to wait until `calcMaxSpend` will provide valid result

Reply:
Thinking on this, it doesn't work just to add a `waitFor` in the test? It looks like the sendMaxButton is only used in one test. Can you wait for a value in the amount input in the test itself?

Reply:
>  Can you wait for a value in the amount input in the test itself
don't understand what you mean, how it will help here? 🤔

Reply:
I think I confused this with the STX form not realizing what was happing with the `calcMaxSpend` for btc. Not sure I have a better solution than the one you are proposing. 🤔